### ISSUE TITLE
_witnetReadResult is throwing request not solved error.

### ISSUE THREAD

**Issue by justanotherdev89** [witnet/witnet-requests-js#76]
Hi team,

I have written code similar to the price feed example.
Using TruffleSuite and in Klaytn's Baobab network. Code is working fine until I get the requestId with 
(uint256 id, ) = _witnetPostRequest(request);
However, when I try to get 
Witnet.Result memory result = _witnetReadResult(_requestId);
It keeps throwing request not solved error. The following is a transaction hash in baobab network.
https://baobab.scope.klaytn.com/tx/0x06b03cd93828c6948ac9fee9e9a51257c4d416cbb877daac43df8d5c777e3e83?tabId=internalTx

I'm using baobab witnet request board which is '0x58D8ECe142c60f5707594a7C1D90e46eAE5AF431'. I checked other configs too but couldn't locate anything noticeable.

Following is my completeUpdate code. I commented out the rest except the first line.
Basically _witnetReadResult just throws me the error.

`    function completeUpdate() public witnetRequestSolved(requestId) {
        Witnet.Result memory result = _witnetReadResult(requestId);

        // if (witnet.isOk(result)) {
        //     string[] memory arr = witnet.asStringArray(result);
        //     Metadata memory arr2 = extractPoseMood(arr);
        // } else {
        //    string memory errorMessage;

        //     try witnet.asErrorMessage(result) returns (Witnet.ErrorCodes, string memory e) {
        //         errorMessage = e;
        //     }
        //     catch (bytes memory errorBytes){
        //         errorMessage = string(errorBytes);
        //     }
        // }
    }`

**Comment by justanotherdev89** [comment]
It actually just throws me error no matter what request I try to read from.
